src/parse.rs

```diff
@@ -1287,6 +1309,7 @@ impl<'t, 'py> Parser<'t, 'py> {
                 parts,
             }),
             "include" => Either::Left(self.parse_include(at, parts)?),
+            "lorem" => Either::Left(TokenTree::Tag(Tag::Lorem(lex_lorem(self.template, parts)?))),
```

Comment:
We'll need a parsing step in addition to the lexing step. For example, `count` can be a variable or an integer and we'll need to handle that here.